stock_aggregated_move_line/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,4 @@
+- Open Inventory > Operations > Transfers and create a new Transfer to Deco Addict
+- Enter two product lines with the same product - Pedal Bin, they will get combined into a single product line
+- Confirm the delivery order and press Print > Delivery Slip
+- The Delivery Slip must also have the two product lines aggregated into one
```

Comment:
And this it show the product description?

The aggregation is not a feature of this module.

Reply:
@janikvonrotz If the product description is different from the default one, in the delivery slip it will be shown under the product name, but not as a separate field

Reply:
@laszlokr My point was that you should test if the product description is shown and not if the aggregation of the product lines works.

Can you update the test and name the exact product description.